generated/nidcpower/nidcpower/session.py

```diff
@@ -7551,6 +7551,28 @@ def _initialize_with_independent_channels(self, resource_name, reset, option_str
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return int(vi_ctype.value)
 
+    @ivi_synchronized
+    def get_channel_names(self, indices):
+        '''get_channel_names
+
+        Returns a list of channel names for the given channel indices.
+
+        Args:
+            indices (basic sequence types or str or int): Index list for the channels in the session. Valid values are from zero to the total number of channels in the session minus one. The index string can be one of the following formats:
+
+                -   A comma-separated list—for example, "0,2,3,1"
```

Comment:
We've had similar debates over repeated capabilities documentation in the past.
Essentially, passing strings and our insane syntax (this function is not as insane but still...) is the only way to use this in C, CVI, C#, LabVIEW.

But in Python we support (and it needs to be in the tests) passing any iterable. This is the most pythonic way of using this so it's the one we should be promoting.

Since this is a change in metadata that might have to be done internally at NI, I'm ok if you follow up with an issue so we can merge this sooner and unblock further work.

Reply:
Upon more detailed reader, I think it's fine. I don't like the emphasis given to the string stuff but I can live with it.

Reply:
Ya, I think the method supports any iterable of int as-is. I shall mark this comment as resolved, feel free to reopen it if there is any further action item on it.